builtin/providers/azurerm/import_arm_container_registry_test.go

```diff
@@ -0,0 +1,61 @@
+package azurerm
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+)
+
+func TestAccAzureRMContainerRegistry_importBasic(t *testing.T) {
+	resourceName := "azurerm_container_registry.test"
+
+	ri := acctest.RandInt()
+	rs := acctest.RandString(4)
+	config := fmt.Sprintf(testAccAzureRMContainerRegistry_basic, ri, rs, ri)
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testCheckAzureRMContainerRegistryDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: config,
+			},
+
+			{
+				ResourceName:            resourceName,
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"storage_account"},
```

Comment:
This is because the inner `access_key` field isn't returned through the API after the initial CreateOrUpdate. Potentially we could retrieve it via the Storage Account - but it felt like the wrong thing to do (given it's a completely unrelated object)?